resolve-hostname/setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Lets rename this to something like `resolve-hostname.sh` or `setup-hostname.sh` in the top-level directory